build.gradle

```diff
@@ -377,6 +377,11 @@ project('reactor-core') {
 	doFirst {
 	  println "Additional tests from `testStaticInit` ($includes)"
 	}
+	afterSuite { TestDescriptor descriptor, TestResult result ->
```

Comment:
the `afterSuite` only applies to this particular test task, correct? (ie. if I exclude the testStaticInit task from the gradle command line, it won't cause any issue)

Reply:
Yes, `afterSuite` is per the `Test` task instance.